.gitignore

```diff
@@ -1,60 +1,41 @@
-# Compiled source #
```

Comment:
These changes to the .gitignore weren't intentional. Looks like I somehow overwrite it with the one I usually use.